R/geom-label.R

```diff
@@ -27,6 +27,10 @@ geom_label <- function(mapping = NULL, data = NULL,
     position <- position_nudge(nudge_x, nudge_y)
   }
 
+  if (!missing(label.size)) {
+    message("`label.size` is deprecated. Please use `linewidth` in the future.")
+  }
```

Comment:
ggplot2 uses the {lifecycle} package to communicate deprecation messages. We'd have to set `label.size = deprecated()` in the formals to have it show up in the usage section of the docs and then use the messaging below:

```suggestion
  if (lifecycle::is_present(label.size)) {
    deprecate_warn0("3.5.0", "geom_label(label.size)", "geom_label(linewidth)")
  }
```

We could also use soft-deprecation with `deprecate_soft0()` here, but then we'd need a fallback mechanism to feed `label.size` into a `linewidth` parameter.

Reply:
Thanks. I couldn't figure out how deprecated messages worked, so I'll trust you on how best to go about it.